gavel/templates/welcome.html

```diff
@@ -1,20 +1,68 @@
-{% extends "layout.html" %}
-{% block title %}Welcome{% endblock %}
-{% block content %}
+{% extends "layout.html" %} {% block title %}Welcome{% endblock %} {% block
+content %}
 <div class="item">
   <div class="banner">
     <h1>Welcome</h1>
   </div>
-  <div class="info">
-    {{ content }}
-  </div>
+  <div class="info">{{ content }}</div>
 </div>
 
 <div class="item">
+  <div class="banner">
+    <h1>Welcome</h1>
+  </div>
+  <div class="info">
+    {{ content }}
+    <div class="info__header">
+      <h1>Hey Javier,</h1>
+    </div>
+
+    <div class="info__guidance">
+      <div class="info__guidance--heading">
+        <h3>How to use Gavel</h3>
+      </div>
+
+      <div class="info__guidance--steps">
+        <div>
+          <h4>First step</h4>
+          <p>
+            You will see one project only to review, when you are done the
+            comparison starts
+          </p>
+        </div>
+        <div>
+          <h4>Compare projects</h4>
+          <p>
+            You will be comparing two projects at a time, from which you have to
+            select one as better than the other
+          </p>
+        </div>
+        <div>
+          <h4>Skipping projects</h4>
+          <p>
+            If a project seems inappropriate, violates event rules or it is not
+            working properly, you can choose the option to skip it
+          </p>
+        </div>
+      </div>
+
+      <div class="info__guidance--footer">
+        <p>
+          If you have problems reviewing the projects, please contact the event
+          organizer
+        </p>
+      </div>
+    </div>
+  </div>
   <div class="info">
     <form action="{{ url_for('welcome_done') }}" method="post">
-      <input type="submit" name="action" value="Continue" class="button-full button-gap positive">
-      <input type="hidden" name="_csrf_token" value="{{ csrf_token() }}">
+      <input
+        type="submit"
+        name="action"
+        value="Start reviewing"
+        class="button-full button-gap positive"
+      />
+      <input type="hidden" name="_csrf_token" value="{{ csrf_token() }}" />
```

Comment:
The  welcome page is not allowing the user to move to start reviewing, I believe is an issue with the value of the input button, not being "continue"

Reply:
As I saw on the Figma, the button display "Start reviewing", should I convert it to "continue"? @wickathou